syntax = "proto3";

import "metadata.proto";

message UR3MoveToJointWaypointRequest {
    int32 waypoint_number = 1;

    RequestMetadata metadata = 100;
}

message UR3MoveToJointWaypointResponse {
    bool success = 1;

    ResponseMetadata metadata = 100;
}